docs/conceptual/l2-cache.rst

```diff
@@ -308,24 +314,22 @@ individual metrics.
 Request flow
 ------------
 
-The following is a diagram that illustrates how L2↔Fabric requests are reported
-by ROCm Compute Profiler:
+The following diagram illustrates how L2↔Fabric requests are reported by ROCm
+Compute Profiler:
 
 .. figure:: ../data/performance-model/fabric.png
    :align: center
    :alt: L2-Fabric transaction flow on AMD Instinct MI-series accelerators
-   :width: 800
```

Comment:
last I checked this, it was pretty illegible at smaller widths, but I have to check again